dummy_robot/dummy_robot_bringup/launch/dummy_robot_bringup_launch.yaml

```diff
@@ -0,0 +1,18 @@
+launch:
+- node:
+    pkg: dummy_map_server
+    exec: dummy_map_server
+    output: screen
+- node:
+    pkg: robot_state_publisher
+    exec: robot_state_publisher
+    output: screen
+    args: $(find-pkg-share dummy_robot_bringup)/launch/single_rrbot.urdf
```

Comment:
@p-vega this form of `robot_state_publisher` configuration [is deprecated](https://github.com/ros/robot_state_publisher/blob/5b3900eb554589b6bf1499cbca3ec35b9899567f/src/robot_state_publisher.cpp#L89-L93) -- use parameters instead of command line arguments, like in the XML.

Reply:
@hidmic Thanks for reviewing, I couldn't find any example of the proper parameter syntax for the yaml launch script. Can you point me in the right direction?

Reply:
Sure, check [this test](https://github.com/ros2/launch_ros/blob/master/test_launch_ros/test/test_launch_ros/frontend/test_node_frontend.py).